CHANGELOG.md

```diff
@@ -8,7 +8,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added|Changed|Deprecated|Removed|Fixed|Security
 Nothing so far
 
-## 4.1.1 - 2020-02-11
+## 4.1.2 - 2020-16-11
+### Changed
+- Introduced the missing dependency on `knplabs/knp-menu-bundle` and updated existing code to be compatible with version 3.
+
+## 4.1.1 - 2020-11-02
```

Comment:
Is 4.1.1 opeens in november gereleased i.p.v. februari?

Reply:
Ja een correctie, hij is op 2 november gereleased.